example/spec/parallel/startSF/startSfSpec.js

```diff
@@ -141,15 +168,14 @@ describe('the sf-starter lambda function', () => {
     });
 
     it('consumes the messages', async () => {
-      const response = await lambda().invoke({
+      const { Payload } = await lambda().invoke({
         FunctionName: sfStarterName,
         InvocationType: 'RequestResponse',
         Payload: JSON.stringify({
           queueUrl,
           messageLimit: testMessageLimit
         })
       }).promise();
-      const { Payload } = response;
       messagesConsumed = parseInt(Payload, 10);
       expect(messagesConsumed).toBeGreaterThan(0);
```

Comment:
I don't understand why this expression is checking for `> 0`. Should this be `>= maxExecutions`?

Reply:
This test is not for a queue with a maximum number of executions defined, but a regular queue. So the amount of messages it would consume (up to the message limit passed to the Lambda) is unpredictable.

Reply:
:doh: good deal.